### PR TITLE
using new pre-processing function from epireview for meta-analysis

### PR DESCRIPTION
**Pull request by annecori**
must be reviewed together with the corresponding PR from epireview:
https://github.com/mrc-ide/epireview/pull/40

### CODE REVIEW

File: src/lassa_delays/orderly.R
Comment (cm401) [inline src/lassa_delays/orderly.R:1]: if we use the develop branch as a staging area (and only push to main infrequently) how can we check that priority pathogens installs from the develop branch? Don't know if we can express this programmatically or just need to make clear on the introduction readme of priority pathogens

**Review by cm401 — APPROVED** [review]
For existing functionality this looks good!

File: src/lassa_delays/orderly.R
Comment (annecori) [reply src/lassa_delays/orderly.R:1]: ah yes excellent point. I can add this into the error message when epireview is installed with a too old version

**Review by sangeetabhatia03 — APPROVED** [review]
Looks good

**Pull request merged**